app/scripts/modules/amazon/serverGroup/details/scalingPolicy/chart/metricAlarmChart.component.spec.js

```diff
@@ -4,7 +4,7 @@ import {Subject} from 'rxjs';
 
 describe('Component: metricAlarmChart', function () {
 
-  var $ctrl, $scope, cloudMetricsReader, $q;
+  var $ctrl, $scope, cloudMetricsReader, rx, $q;
```

Comment:
ok, i think this was just an errant non-deletion.  this was the only case.  fixed.